src/components/claiming/Countdown.vue

```diff
@@ -150,15 +149,13 @@ export default {
     .text {
       color: $font-color-light;
       margin-top: 24px;
-      font-size: 20px;
       margin-bottom: 16px;
       text-align: center;
     }
 
     .digit {
-      color: $font-color-light;
       font-size: 50px;
-      font-weight: 700px;
+      font-weight: 400px;
```

Comment:
`font-weight` doesn't accept any unit. On the other hand, I this this has no effect.
```suggestion
```